src/plugins/clips-executive/clips/goal-tree.clp

```diff
@@ -54,3 +54,35 @@
 	(goal-tree-update-child ?fact-address ?id 0)
 	(return ?goal)
 )
+
+(deffunction goal-tree-assert-run-parallel (?class ?continue-on $?fact-addresses)
```

Comment:
Please document the function, especially its arguments

Reply:
done